shared/oval/sshd_use_approved_ciphers.xml

```diff
@@ -4,12 +4,12 @@
       <title>Use Only Approved Ciphers</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_debian</platform>
       </affected>
       <description>Limit the ciphers to those which are FIPS-approved and only
       use ciphers in counter (CTR) mode.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
```

Comment:
I don't know what the past path is and need help sounding this out and thinking through it.

Starting with _intent_:
Most (all?) regulated industries require evaluated cryptography. In the US Gov, HIPAA, and PCI areas, this means formal FIPS 140-2 evaluations. Currently non-RHEL systems (Fedora, Cent, Debian) simply do not have FIPS evaluations.

Debian and CentOS users certainly want to use strong crypto -- even if they aren't formally certified. However even if sshd is configured against certain Ciphers and MACs on non-RHEL, the system can not be in compliance with using FIPS crypto. That's why certain checks (like this one) have the installed_OS_is_certified extended definition attached to it.

If we remove ``installed_OS_is_certified criterion``,  STIG or PCI scans on non-RHEL will have false positives. Renaming the rule and keeping certified/approved/evaluated wording would have the same issue (non-RHEL ciphers are neither certified or approved). Both seem like blockers to me.

Copy/pasting existing content into a new rule (e.g. ``sshd_use_strong_ciphers``?), remove installed_OS_is_certified, and adding a note that while Ciphers are used they are not approved/tested/validated, does address the above concerns. But something still feels wrong about that approach and I'm having a hard time articulating it.

IMHO, non-RHEL systems should *always* fail these kinds of checks. But I'll self-acknowledge my bias towards systems that must meet formal compliance regimes (often based in legal requirements). Commercial system owners may not care about such evaluations.

If we create new XCCDF and use platform tags to have different language for RHEL vs non-RHEL, we still need a way to show non-RHEL systems fail formal requirements in baselines like PCI and anything Government related. So having a single XCCDF with different prose does not seem like a solution. Everything points to creating a net-new rule. I just don't know how to phrase the XCCDF.